test/built-ins/RegExp/prototype/source/prop-desc.js

```diff
@@ -2,16 +2,19 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+esid: sec-get-regexp.prototype.source
 es5id: 15.10.7.1-2
+es6id: 21.2.5.10
 description: >
     RegExp.prototype.source is an accessor property whose set accessor
     function is undefined
+includes: [propertyHelper.js]
 ---*/
 
-  var d = Object.getOwnPropertyDescriptor(RegExp.prototype, 'source');
-  
+var d = Object.getOwnPropertyDescriptor(RegExp.prototype, 'source');
 
 assert.sameValue(typeof d.get, 'function', 'typeof d.get');
 assert.sameValue(d.set, undefined, 'd.set');
-assert.sameValue(d.enumerable, false, 'd.enumerable');
-assert.sameValue(d.configurable, true, 'd.configurable');
+
+verifyNotEnumerable(RegExp.prototype, 'source');
+verifyConfigurable(RegExp.prototype, 'source');
```

Comment:
why not using `verifyProperty` here? For accessor descriptors, you can do this:

```js
verifyProperty(RegExp.prototype, 'source', {
  enumerable: false,
  configurable: true,
});
```

There's no support to compare `get` and `set` yet but the other parts can already use it.